classes/Model/Theme.php

```diff
@@ -439,7 +439,7 @@ public function setup( $post ) {
 			return;
 		}
 
-		if ( $this->ID === 5 ) {
+		if ( $this->ID === 5 ) {  // phpcs:ignore
```

Comment:
@s-acv2 - Sensing a pattern. Don't be scared of object notation.

Just realize that `$var['ID']` is array notation in PHP and `$var->ID` is object notation. Both represent a pointer to the ID property of the $var variable, but access method depends on type.

So `5 === $this->ID` , the ->ID is again part of the reference to the variable being compared.